homeassistant/util/dt.py

```diff
@@ -92,6 +95,10 @@ def get_time_zone(time_zone_str: str) -> dt.tzinfo | None:
     try:
         return zoneinfo.ZoneInfo(time_zone_str)
     except zoneinfo.ZoneInfoNotFoundError:
+        try:
+            import tzdata
+        except ImportError:
+            _LOGGER.warn('Timezone %s not found. Try installing tzdata package and retry.', time_zone_str)
```

Comment:
Native Windows is not a supported installation OS, and we've been removing workarounds for it.

As far as I know, zoneinfo is natively supported on the main Linux distributions and Mac OS.

Reply:
Alright I got it. But to some degree I guess it does not hurt to add workarounds if, by any chance, `zoneinfo` is not working properly.

Reply:
We'd rather not add any workarounds to keep maintenance burden and the risk that something breaks as low as possible.